bin/node/runtime/src/lib.rs

```diff
@@ -495,6 +495,10 @@ impl_opaque_keys! {
 	}
 }
 
+parameter_types! {
+	pub const ValidatorSet: u32 = 3072;
```

Comment:
`MaxValidators` makes more sense to me, WDYT?

Reply:
LGTM!